src/history/history-detail-overview.component.tsx

```diff
@@ -0,0 +1,64 @@
+import React  from 'react';
+import { InlineLoading, Tabs, TabPanel, TabPanels } from '@carbon/react';
+import { formatDatetime, parseDate, ExtensionSlot, ErrorState } from '@openmrs/esm-framework';
+import parse from 'html-react-parser';
+import { type ReactI18NextChild, useTranslation } from 'react-i18next';
+import useIpsResource from '../common/history.resource';
+import styles from './history-detail-overview.scss';
+import { EmptyState } from '../empty-state';
+
+interface HistoryDetailOverviewProps {
+  patientUuid: string;
+}
+
+const HistoryDetailOverview: React.FC<HistoryDetailOverviewProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const { summaries, error, isLoading } = useIpsResource();
+
+  return (
+    <div className={styles.tabs}>
+      <Tabs>
+        <TabPanels>
+          <TabPanel>
+            {isLoading ? (
+              <InlineLoading description={`${t('loading', 'Loading')} ...`} role="progressbar" />
+            ) : error ? (
+              <ErrorState headerTitle={t('history', 'Patient History')} error={error} />
+            ) : summaries?.entry ? (
+              <>
+                {summaries.entry.map((summary: { resource: { resourceType: string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | Iterable<React.ReactNode> | React.ReactPortal | Iterable<ReactI18NextChild>; effectiveDateTime: string; text: { div: any; }; }; }, i: React.Key) => (
```

Comment:
It would probably be better if `useIpsResource()` set the type correctly rather than doing it here.